nimsuggest/nimsuggest.nim

```diff
@@ -207,6 +207,16 @@ proc listEpc(): SexpNode =
     methodDesc.add(docstring)
     result.add(methodDesc)
 
+proc runGc() =
+  ## Runs a GC collection pass. This procedure will also collect any cycles assuming ORC is in use.
+  ##
+  ## When nimsuggest is being used as a library, no GC mode change will be performed.
+  when isMainModule and defined(gcOrc):
+    GC_enableOrc()
+  GC_fullCollect()
+  when isMainModule and defined(gcOrc):
+    GC_disableOrc()
```

Comment:
@zerbina do you know if I need to toggle orc state to run the cycle collector?

Reply:
Enabling ORC prior to `GC_fullCollect` is not necessary, but if you want to keep the cycle collector disabled afterwards, the `GC_disableOrc` is needed.